webpack/client.js

```diff
@@ -1,12 +1,23 @@
+/* eslint import/no-extraneous-dependencies: 0 */
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
 const paths = require('./paths');
 const baseConfig = require('./base');
+const sassRules = require('./sassRules');
 
 const clientConfig = {
   ...baseConfig,
   name: 'client',
   entry: {
     app: paths.entries.client,
   },
+  module: {
+    ...baseConfig.module,
+    rules: [
+      ...baseConfig.module.rules,
+      sassRules.client,
+    ],
+  },
```

Comment:
It may be beneficial to separate out `modules` config in the future and distinguish between client and server configs for dev and prod.